website/content/api-docs/client.mdx

```diff
@@ -20,15 +20,190 @@ When accessing the endpoints via the server, if the desired node is ambiguous
 based on the URL, an additional `?node_id` query parameter must be provided to
 disambiguate.
 
+## Read Node Metadata
+
+This endpoint queries Node metadata on a specific Client agent and responds
+with the following fields:
+
+- `Meta` `(object)` - The Node metadata that will be registered with the Nomad
+  servers and used by the scheduler (after up to 10 seconds of delay for
+  batching). This is the merged version of the `Static` and `Dynamic` fields.
+
+- `Static` `(object)` - The Node metadata set in the Client agent's
+  configuration file. Only loaded when an agent starts.
+
+- `Dynamic` `(object)` - The Node metadata set via the API (see below). Unlike
+  `Meta` and `Static`, this object may contain `null` values to differentiate
+  "unset" keys from keys with an empty string value (`""`).
+
+Note that [`/v1/node/:node_id`][api-node-read] only contains the `Meta` object.
+It may take up to 10 seconds for dynamic Node metadata to be sent to Servers
+and visible through the Node API. Use the Node API to see the version of Node
+metadata the scheduler uses.
+
+| Method | Path                  | Produces           |
+| ------ | --------------------- | ------------------ |
+| `GET`  | `/v1/client/metadata` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/nomad/api-docs#blocking-queries) and
+[required ACLs](/nomad/api-docs#acls).
+
+| Blocking Queries | ACL Required  |
+| ---------------- | ------------- |
+| `NO`             | `node:read`   |
+
+### Parameters
+
+- `:node_id` `(string: <optional>)` - Specifies the node to query.
+  This is required when the endpoint is being accessed via a server. Defaults
+  to the node recieving the request otherwise. This is specified as part of
+  the URL.  Note, this must be the _full_ node ID, not the short 8-character
+  one. This must be specified as part of the path (`?node_id=...`).
+
+### Sample Request
+
+```shell-session
+$ nomad operator api /v1/client/metadata
+```
+
+```shell-session
+$ curl http://127.0.0.1:4646/v1/client/metadata
+```
```

Comment:
probably unnecessary but it doesn't hurt (we wouldn't want curl alternatives in every case we use opapi in an example?)

Reply:
We took a vote internally:

nomad operator api - :six: 
curl - :five: 

I'll change this page's examples but not do a big old s/curl/noa/g in this PR